src/backend/src/services/projects.services.ts

```diff
@@ -811,4 +811,87 @@ export default class ProjectsService {
 
     return newMaterialType;
   }
+
+  /**
+   * Update a material
+   * @param name the name of the new material
+   * @param assembly the assembly of the new material
+   * @param status the Material status of the new material
+   * @param manufacturerName the manufacture name of the new material
+   * @param manufacturerPartNumber the manufacture part number of the new material
+   * @param pdmFileName the pdm file name of the new material
+   * @param quantity the quantity of the new material
+   * @param quantityUnit the quantity unit of the new material
+   * @param unitName the unit name of the new material
+   * @param price the price of the new material
+   * @param subtotal the subtotal of the new material
+   * @param linkUrl the link url of the new material
+   * @param notes the notes of the new material
+   */
+  static async editMaterial(
+    submitter: User,
+    wbsNumber: WbsNumber,
+    name: string,
+    status: Material_Status,
+    manufacturerName: string,
+    manufacturerPartNumber: string,
+    quantity: number,
+    unitName: string,
```

Comment:
should optional, this is what you use to connect the quantity unit, it's a bit confusing name